docs/circuit_cutting/explanation/index.rst

```diff
@@ -10,39 +10,185 @@ Circuit cutting is a technique to increase the size of circuits we can run on qu
 
-There are three settings to consider for circuit cutting.  The first is where only local operations (LO) [i.e., local *quantum* operations] are available.  The other settings introduce classical communication between the circuit executions, which is known in the quantum information literature as LOCC, for `local operations and classical communication <https://en.wikipedia.org/wiki/LOCC>`__.  The LOCC can be either near-time, one-directional communication between the circuit executions (the second setting), or real-time, bi-directional communication (the third setting).
+There are `three settings <https://research.ibm.com/blog/circuit-knitting-with-classical-communication>`__ to consider for circuit cutting.  The first is where only local operations (LO) [i.e., local *quantum* operations] are available.  The other settings introduce classical communication between the circuit executions, which is known in the quantum information literature as LOCC, for `local operations and classical communication <https://en.wikipedia.org/wiki/LOCC>`__.  The LOCC can be either near-time, one-directional communication between the circuit executions (the second setting), or real-time, bi-directional communication (the third setting).
 
-As mentioned above, the cost of any simulation based on quasiprobability distribution is an exponential sampling overhead. The overhead of a cut gate depends on which gate is cut; see the final appendix of [`1 <https://arxiv.org/abs/2205.00016>`__] for details.  Here, we will focus on the CNOT gate.  If no real-time classical communication is available between qubits of the cut gate or wire, cut CNOT gates incur a sampling overhead of O(:math:`9^n`), and wire cuts incur a sampling overhead of O(:math:`16^n`), where :math:`n` is the total number of cuts. If real-time communication is available (i.e., if the hardware supports “dynamic circuits”), the sampling overhead for both CNOT gate and wire cuts may be reduced to O(:math:`4^n`) [`1 <https://arxiv.org/abs/2205.00016>`__,\ `4 <https://arxiv.org/abs/2302.03366>`__]; however, support for circuit cutting with classical communication (LOCC) is not yet supported in CKT.
+As mentioned above, the cost of any simulation based on quasiprobability distribution is an exponential sampling overhead.
+The sampling overhead is the factor by which the overall number of shots must increase for the quasiprobability decomposition to result in the same amount of error, :math:`\epsilon`, as one would get by executing the original circuit.
```

Comment:
Isn't there a sampling error we must incur due to this technique, and the sampling drives the sampling error down to epsilon, which approaches but never reaches 0?

Is it more accurate to say ... "same amount of error as one would get by executing the original circuit, within some tolerance epsilon"?

Reply:
There is always sampling error because we can only ever take a finite number of shots on real (or even ideal) hardware.  The errors get exaggerated during QPD post-processing by the factor $\gamma$ (see below in the document, "This post-processing boosts the magnitude of each measurement outcome ...").  To keep the error the same, one must multiply the number of shots by $\gamma^2$ to keep the standard error ($\epsilon/\sqrt{n}_{shots}$) constant.

I added one word, but do you think this clarifies anything?

```suggestion
The sampling overhead is the factor by which the overall number of shots must increase for the quasiprobability decomposition to result in the same amount of sampling error, :math:`\epsilon`, as one would get by executing the original circuit.
```

Reply:
>To keep the error the same, one must multiply the number of shots by $\gamma^{2}$
 to keep the standard error...
 
 Based on this, it sounds like it's the standard error $(\epsilon/\sqrt{n})$ that is being kept the same, rather than the sampling error $(\epsilon)$.

Reply:
I see, understood, thanks!